<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureTool</class>
 <widget class="QMainWindow" name="TextureTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>texture tool</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="m_graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="m_tooBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actionOpen"/>
   <addaction name="m_actionSave"/>
   <addaction name="separator"/>
   <addaction name="m_actionR"/>
   <addaction name="m_actionG"/>
   <addaction name="m_actionB"/>
   <addaction name="m_actionA"/>
   <addaction name="separator"/>
   <addaction name="m_actionAddSubTex"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoScrollMargin">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="m_actionOpen">
   <property name="icon">
    <iconset resource="../App.qrc">
     <normaloff>:/icon/Icon/Open.png</normaloff>:/icon/Icon/Open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="m_actionSave">
   <property name="icon">
    <iconset resource="../App.qrc">
     <normaloff>:/icon/Icon/save.png</normaloff>:/icon/Icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="m_actionR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../App.qrc">
     <normaloff>:/icon/Icon/Btn_TextureProperties_Red_On.png</normaloff>:/icon/Icon/Btn_TextureProperties_Red_On.png</iconset>
   </property>
   <property name="text">
    <string>开关红色通道</string>
   </property>
   <property name="toolTip">
    <string>开关红色通道</string>
   </property>
  </action>
  <action name="m_actionG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../App.qrc">
     <normaloff>:/icon/Icon/Btn_TextureProperties_Green_On.png</normaloff>:/icon/Icon/Btn_TextureProperties_Green_On.png</iconset>
   </property>
   <property name="text">
    <string>开关绿色通道</string>
   </property>
   <property name="toolTip">
    <string>开关绿色通道</string>
   </property>
  </action>
  <action name="m_actionB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../App.qrc">
     <normaloff>:/icon/Icon/Btn_TextureProperties_Blue_On.png</normaloff>:/icon/Icon/Btn_TextureProperties_Blue_On.png</iconset>
   </property>
   <property name="text">
    <string>开关蓝色通道</string>
   </property>
   <property name="toolTip">
    <string>开关蓝色通道</string>
   </property>
  </action>
  <action name="m_actionA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../App.qrc">
     <normaloff>:/icon/Icon/Btn_TextureProperties_Alpha_On.png</normaloff>:/icon/Icon/Btn_TextureProperties_Alpha_On.png</iconset>
   </property>
   <property name="text">
    <string>开关Alpha通道</string>
   </property>
   <property name="toolTip">
    <string>开关Alpha通道</string>
   </property>
  </action>
  <action name="m_actionAddSubTex">
   <property name="icon">
    <iconset resource="../App.qrc">
     <normaloff>:/icon/Icon/addPart.png</normaloff>:/icon/Icon/addPart.png</iconset>
   </property>
   <property name="text">
    <string>添加子纹理</string>
   </property>
   <property name="toolTip">
    <string>添加子纹理</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../App.qrc"/>
 </resources>
 <connections/>
</ui>
